webpack.config.js

```diff
@@ -43,6 +43,22 @@ module.exports = (env = {}) => {
         info.resourcePath.replace(/^\.\//, ''),
       crossOriginLoading: 'anonymous',
     },
+    optimization: {
+      splitChunks: {
+        cacheGroups: {
+          commons: {
+            name: 'vendor',
+            test: /[\\/]node_modules[\\/]/,
+            chunks: 'initial',
+          },
+          commons2: {
```

Comment:
So what does `commons2` do? I ran analyze with and without it and don't see any difference...

Reply:
Yeah, you're right. It does not seem to affect anything either way. I had brought it over initially because there was a CommonChunksPlugin for it.  I'll go ahead and remove it just for simplicity sake. :+1: